packages/main/src/Link.js

```diff
@@ -84,17 +85,20 @@ const metadata = {
 		},
 
 		/**
-		 * Defines whether the component text should wrap
-		 * when there is no sufficient space.
-		 * <br><br>
-		 * <b>Note:</b> The text is truncated by default.
+		 * Defines how the text of a component will be displayed when there is not enough space.
+		 * Available options are:
+		 * <ul>
+		 * <li><code>None</code> - The text will be truncated with an ellipsis.</li>
+		 * <li><code>Normal</code> - The text will wrap. The words will not be broken based on hyphenation.</li>
+		 * </ul>
 		 *
-		 * @type {boolean}
-		 * @defaultvalue false
+		 * @type {string}
```

Comment:
Yes, this becomes a little odd, the type in the JSDoc should be changed
@type {string} -> @type {WrappingType} and both the type and the name of the prop will become "WrappingType":
```js
/**
@type {WrappingType}
*/
 wrappingType: {
			type: WrappingType,
			defaultValue: WrappingType.None,
		},
```

Not sure again, maybe we have to consider naming the enum "Wrapping".